### PR TITLE
Added apt-get update to Resynchronize the package index files

### PR DESCRIPTION
**Pull request by sandoche2k**

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -57,13 +57,19 @@ This instruction covers the following operating systems:
 
 ### Installation on Debian
 
-1) Install dependencies from binary packages:
+1) Resynchronize the package index files
+
+  ```sh
+  sudo apt-get update
```
Comment (matsduf) [inline docs/Installation.md]: Please indent so that i aligns with the command below.

Comment (pawal) [reply docs/Installation.md]: "Refresh package information" is more in line with the intention of "apt-get update".

**Comment by sandoche2k** [comment]
@matsduf please merge

**Comment by matsduf** [comment]
@sandoche2k, I have approved, so I suggest that you merge.

**Pull request merged**